src/toniarts/openkeeper/tools/convert/map/Thing.java

```diff
@@ -16,11 +16,13 @@
  */
 package toniarts.openkeeper.tools.convert.map;
 
+import com.jme3.network.serializing.serializers.EnumSerializer;

```

Comment:
Yeah.. I guess this is ok for now, since it was already misused. But jME or network classes don't belong here. Same hygiene stuff.

Reply:
Agree